velox/vector/BaseVector.h

```diff
@@ -428,13 +436,14 @@ class BaseVector {
   }
 
   // Sets null when 'nulls' has a null value for active rows in 'rows'.
-  // Is a no-op 'nulls' is a nullptr or 'rows' has no selections.
+  // Is a no-op 'nulls' is a nullptr or 'rows' has no selections. This API is
+  // not supported for ConstantVector.
```

Comment:
what happens in these cases? Does it throw?

Reply:
Yes, this API throws if the vector is a ConstantVector. I've updated the comment to say so. Thank you for the suggestion!